src/DataTypes/DataTypeFixedString.h

```diff
@@ -42,6 +42,7 @@ class DataTypeFixedString final : public IDataType
     TypeIndex getTypeId() const override { return type_id; }
 
     const char * getFamilyName() const override { return "FixedString"; }
+    const char * getSQLCompatibleName() const override { return "TEXT"; }
```

Comment:
Isn't that `CHAR` in MySQL? https://dev.mysql.com/doc/refman/8.0/en/char.html

EDIT: Ah, guess TEXT is more fitting because it can hold arbitrary bytes and CHAR can't. Maybe add a comment like in DataTypeString.h.